src/middlewared/middlewared/plugins/disk_/wipe.py

```diff
@@ -91,8 +91,10 @@ def _wipe_impl(self, job, dev, mode, event):
 
                 # The middle partitions often contain old cruft.  Clean those.
                 if len(disk_parts) > 1:
-                    for partnum, sector_start in disk_parts.items():
-                        if partnum == 1:
+                    _30MiB = 30 * CHUNK
+                    for sector_start in disk_parts.values():
+                        # Skip any partitions that start under 30 MiB
+                        if sector_start < _30MiB:
```

Comment:
Should we also skip any partitions that are in the last 32MB of the drive?  (I know only those starting less than 4MB before the end would cause a problem.)